config/helm/chart/default/values.yaml

```
@@ -194,3 +194,16 @@ csidriver:
         cpu: 20m
         memory: 30Mi
 
+rbac:
+  activeGate:
+    annotations:
+    kubernetesMonitoring:
+      annotations:
```

would it make sense to have a separate section for the "base" `activeGate` ServiceAccount ?
because currently, it is kinda confusing. 
It could either work like:
- `rbac.activeGate.annotations` be on all `activeGate` ServiceAccounts and `rbac.activeGate.kubernetesMonitoring.annotation` would overrule it IF defined
- both `annotations` sections are handled independently are totally separate


```suggestion
  activeGate:
     default: # horrible name, I know
    	annotations:
    kubernetesMonitoring:
      annotations:
```

---

I think it is best if we keep the structure as is, but just implement the override logic. 

So the KubernetesMonitoring sa always uses the ActiveGate if not defined.

Not really a fan of the default section

---

me neither
the problem kinda comes from the fact that the "default" AG serviceAccount has 0 roles assigned to it :) 
as only the `kubernetesMonitoring` capability is interacting with the kube-api

---

btw: the override logic will not make it better, probably worse 😅

---

Hmm, then lets separate them completely. 🤔

---

does it even make sense to separate the `annotations`? You'll use either of the service accounts, but both should have the same annotations.

So keep `rbac.activegate.annotations` and put it on activegate and kubernetes monitoring service account.

---

Done